source/features/latest-tag-button.tsx

```diff
@@ -18,45 +30,83 @@ const getLatestTag = cache.function(async (): Promise<string | false> => {
 			}) {
 				nodes {
 					name
+					target {
+						oid
+						... on Commit {
+							committedDate
+						}
+					}
 				}
 			}
 		}
 	`);
 
-	const tags: string[] = repository.refs.nodes.map((tag: {name: string}) => tag.name);
+	const tags: TagInfo[] = repository.refs.nodes.map((tag: {name: string; target: {oid: string; committedDate: string}}) => new TagInfo(tag));
```

Comment:
Use regular objects, not classes, they can’t be saved in cache. But in this case this change isn’t needed at all because dates shouldn’t be compared.